### PR TITLE
fix: feedback diglog position

### PR DESCRIPTION
**Pull request by AhyoungRyu**
**AS-IS**
Screenshot 2024-04-22 at 2 30 07 PM

**TO-BE**
The dialog should be attached to the bottom of widget window element not the entire DOM. 
Screenshot 2024-04-22 at 2 19 10 PM

### CODE REVIEW

File: src/hooks/useDynamicAttachModal.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                parent.appendChild(child);
+              }
+            }
+          });
+        }
+      });
+    });
+
+    // Set the observer to monitor the entire document
+    observer.observe(document.body, { childList: true, subtree: true });
```
Comment (bang9) [inline src/hooks/useDynamicAttachModal.tsx:31]: If attached to the body, it seems there may be overhead detecting changes to the entire DOM unrelated to the widget. How about testing it by attaching it to 'sendbird-modal-root'?

Comment (bang9) [reply src/hooks/useDynamicAttachModal.tsx:31]: Oh, so it's not about attaching individual modals added but moving the entire modal root under the widget window.

Comment (AhyoungRyu) [reply src/hooks/useDynamicAttachModal.tsx:31]: Yeah that's correct.

Comment (bang9) [reply src/hooks/useDynamicAttachModal.tsx:31]: After moving the modal root, if you disconnect it, will the modal root return to its original position during the next rendering?

Comment (AhyoungRyu) [reply src/hooks/useDynamicAttachModal.tsx:31]: Hmm it'd be better if there's a way to get a root element of this widget module than using `document.body`. 

>After moving the modal root, if you disconnect it, will the modal root return to its original position during the next rendering?

No it won't. It'll be under `id: aichatbot-widget-window` element once it's moved. Do you think it should be put back to its original position? @bang9

**Review by bang9 — APPROVED** [review]

File: src/hooks/useDynamicAttachModal.tsx (hunk 1 above)
Comment (AhyoungRyu) [reply src/hooks/useDynamicAttachModal.tsx:31]: @bang9 As we discussed in Slack, I changed it to disconnect right after it moves the modal root element to the widget window element. bd19db95ba94bd705f01d2d9b2832cc4cf99e93f
Also, it's working as expected in both mobile & desktop view. 

- Desktop
Screenshot 2024-04-22 at 5 46 52 PM

- Mobile
Screenshot 2024-04-22 at 5 47 07 PM

**Pull request merged**